fastlane/Appfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+json_key_file('') # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name('com.nimbl3') # e.g. com.krausefx.app
```

Comment:
I can't recall, but yeah just make sure this package name is mentioned to the README (HOWTO) as well 👍

Reply:
Should I maybe leave this blank instead?

I have blanked SLACK_URL and app_id already.

Reply:
you can use a cryptic name for it perhaps? e.g, `'com.your.package.name'`